examples/go-tcx-counter/container-deployment/Containerfile.go-tcx-counter

```
@@ -0,0 +1,30 @@
+ARG BUILDPLATFORM=linux/amd64
+
+FROM --platform=$BUILDPLATFORM golang:1.22 AS go-tcx-counter-build
+
+ARG BUILDPLATFORM
+
+# The following ARGs are set internally by docker or podman on multiarch builds
+ARG TARGETPLATFORM=linux/amd64
```

```suggestion
ARG TARGETARCH
ARG TARGETOS
ARG TARGETPLATFORM

RUN echo "TARGETOS=${TARGETOS}  TARGETARCH=${TARGETARCH}  BUILDPLATFORM=${BUILDPLATFORM}  TARGETPLATFORM=${TARGETPLATFORM}"
```
Similar change made in all the files in the multi-arch PR.

---

Done.